dhcpv4/nclient4/client.go

```diff
@@ -524,6 +524,35 @@ func (c *Client) RequestFromOffer(ctx context.Context, offer *dhcpv4.DHCPv4, mod
 	return lease, nil
 }
 
+// InformFromOffer sends an Inform request and waits for a response.
+func (c *Client) InformFromOffer(ctx context.Context, offer *dhcpv4.DHCPv4, modifiers ...dhcpv4.Modifier) (*dhcpv4.DHCPv4, error) {
```

Comment:
This looks like a copy of `RequestFromOffer` except it is using `NewInformFromOffer`.
Can it be abstracted out to reuse code?

Reply:
It should be possible to reuse the `c.SendAndRead` part in the middle, so I'll do some refactoring.

Reply:
The functionality has now been refactored into `TransactAckNak`.